content/_data/upgrades/2-235-1.adoc

```diff
@@ -0,0 +1,23 @@
+==== Plugin manager improvements
```

Comment:
It is unclear why this is in the upgrade guide.

In the past we've used the upgrade guide to inform administrators about changes they need to know, typically breaking changes of some sort. Often, changes that we add escape hatches for would be included, in case they find themselves in need of them.

While I would hope the UI makes it clear that the Available tab no longer lists issues by default, that's the one aspect of the plugin manager changes that could make sense to call out. Otherwise, not really IMO.

Reply:
I've reduced to a shorter phrasing that focuses on the change in the "Available" tab.  I want it as a location we can reference when someone asks "Why is my 'Available' tab empty?"  That will help them learn that we have an upgrade guide and that it contains focused and useful information for their upgrade.

See e87f714f9ddf03835346f9c0d1dcfbed3a1f8a37

Reply:
Should be fine now.